src/app/modules/billing/billing.controller.ts

```diff
@@ -54,7 +51,7 @@ export const handleGetBillInfo: RequestHandler<
     })
     return res
       .status(StatusCodes.INTERNAL_SERVER_ERROR)
-      .json('Error in retrieving billing records')
```

Comment:
double check to see if frontend client acts on the error message in any way for backwards compatibility.

Reply:
the frontend client only logs the `error` object at most and does not do anything meaningful beyond that